js/sentry.ts

```diff
@@ -0,0 +1,19 @@
+import { getMetaContent } from "./util/metadata";
+import * as Sentry from "@sentry/react";
+
+const sentryDsn = getMetaContent("sentryDsn");
+const sentryEnvironment = getMetaContent("sentryEnvironment");
+const release = getMetaContent("release");
+const userEmail = getMetaContent("userEmail");
+
+if (sentryDsn !== null && sentryEnvironment !== null) {
+  Sentry.init({
+    dsn: sentryDsn,
+    environment: sentryEnvironment,
+    release: release ?? undefined,
+  });
+
+  if (userEmail !== null) {
+    Sentry.setUser({ email: userEmail });
+  }
+}
```

Comment:
**suggestion:** rather than having this code run directly at the top level on import, how about wrapping it in an exported `initializeSentry` function and calling it explicitly from `js/app.tsx`? I personally find that more explicit / easier to reason about.

Reply:
Interestingly they actually recommend this, probably to avoid other global scope code from beating it. https://docs.sentry.io/platforms/javascript/guides/react/

Reply:
Ah, okay - if it's recommended by the docs then I'm fine with it.